### PR TITLE
[new] add leaflet-plugins@1.3.12 with git auto-update

### PR DESCRIPTION
**Pull request by Amomo**
Hi @Piicksarn 

For #6137, I add leaflet-plugins at v1.3.12 and add git auto-update config in its package.json.
repo: https://github.com/shramov/leaflet-plugins

As README.md and bower.json said, I add `Vector layers (layer/vector/)`, `Providers (layer/tile)` , `(layer/)` and `control plugins (control/)`.

This library contains plugins of leaflet, so it don't has a main file.
Randomly, I pick "layer/vector/OSM.min.js" for filename field.

Would you mind checking this PR for me? Thank you~ :-D

- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity : **Watch `54`, Star `440`, Fork `206`**
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.

### CODE REVIEW

**Comment by Piicksarn** [comment]
It's fine!
@PeterDaveHello, would you please have a look at the pr?

the main files shown in bower.json :
```
"main": [
    "control/Distance.js",
    "control/Layers.Load.js", 
    "control/Permalink.js",
    "control/Permalink.Layer.js",
    "control/Permalink.Line.js",
    "control/Permalink.Marker.js",
    "layer/Icon.Canvas.js",
    "layer/Layer.Deferred.js",
    "layer/Marker.Rotate.js",
    "layer/Marker.Text.js",
    "layer/OpenStreetBugs.js",
    "layer/vector/GPX.js",
    "layer/vector/GPX.Speed.js",
    "layer/vector/KML.js",
    "layer/vector/OSM.js",
    "layer/tile/Bing.js",
    "layer/tile/Google.js",
    "layer/tile/Yandex.js"
]
```

**Comment by PeterDaveHello** [comment]
hmmmm ... I would like to ask author to confirm that.

@shramov would you please give us some advice if we wanna host this lib on the cdn? Thanks.

**Comment by Amomo** [comment]
Hi @brunob would you please give us some advice if we wanna host this lib on the cdn?
Which files should we add? Is there a main file we could point to?
Thank you! :)

**Comment by brunob** [comment]
@Amomo there is no main file, each script is independent, the scripts listed in bower file seems ok for me.

**Comment by Amomo** [comment]
@brunob thanks for your response! :-D

**Comment by Amomo** [comment]
@Piicksarn I have rebased this PR, could you check it again? Thank you!

**Comment by Piicksarn** [comment]
@PeterDaveHello, I think the pr is fine!
Would you mind to have a look at the pr?
Thank you!

**Comment by PeterDaveHello** [comment]
Duplicated file map ...

**Comment by Amomo** [comment]
@PeterDaveHello sorry, I am not sure what does your "Duplicated file map" means.
The library doesn't provide minified files, so I note the needed minified files in "requiredFiles" field.

Or, could you point me which part I did it wrong? Thank you!

File: ajax/libs/leaflet-plugins/package.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    "source": "git",
+    "target": "git://github.com/shramov/leaflet-plugins.git",
+    "basePath": "",
+    "files": [
+      "control/Distance*.js",
+      "control/Layers.Load*.js",
+      "control/Permalink*.js",
+      "control/Permalink.Layer*.js",
+      "control/Permalink.Line*.js",
+      "control/Permalink.Marker*.js",
```
Comment (PeterDaveHello) [inline ajax/libs/leaflet-plugins/package.json]: Doesn't `"control/Permalink*.js"` include `"control/Permalink.Layer*.js"`?

Comment (Amomo) [inline ajax/libs/leaflet-plugins/package.json]: Oh yes! Thanks a lot.

**Comment by Amomo** [comment]
@Piicksarn Would you mind to check this PR for me? Thank you~ :-)

**Comment by PeterDaveHello** [comment]
@LeaYeh @maruilian11 would you please also help review this PR? Thanks.

**Comment by Piicksarn** [comment]
@PeterDaveHello, the pr is fine.
would you please have a look at the pr?
thanks.

**Comment by maruilian11** [comment]
@PeterDaveHello i think the pr is ok.

**Comment by PeterDaveHello** [comment]
2 days ago ...

**Comment by maruilian11** [comment]
@Amomo could you rebase the commits?

**Comment by Amomo** [comment]
@maruilian11 @PeterDaveHello @Piicksarn thank you.
I have rebase on latest master branch.

**Comment by maruilian11** [comment]
@Amomo sorry... could you rebase it again 😞

**Comment by Amomo** [comment]
@maruilian11 I have updated it to v1.4.2, could you help me check this PR again?
Thank you~ :-D

**Comment by maruilian11** [comment]
@Amomo sorry..again😭

**Comment by Amomo** [comment]
@maruilian11 Could you help me check it again? Thank you~ :-D

**Comment by maruilian11** [comment]
@PeterDaveHello i think this PR is ok.

**Comment by PeterDaveHello** [comment]
@Piicksarn @LeaYeh please help double check this, thanks.

**Pull request closed without merging**